docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md

```diff
@@ -95,7 +95,7 @@ The use of asynchronous communication is explained with additional details later
   [*https://docs.microsoft.com/azure/architecture/patterns/materialized-view*](https://docs.microsoft.com/azure/architecture/patterns/materialized-view)
 
 - **Charles Row. ACID vs. BASE: The Shifting pH of Database Transaction Processing** \
-  [*http://www.dataversity.net/acid-vs-base-the-shifting-ph-of-database-transaction-processing/*](http://www.dataversity.net/acid-vs-base-the-shifting-ph-of-database-transaction-processing/)
+  [*https://www.dataversity.net/acid-vs-base-the-shifting-ph-of-database-transaction-processing/*](https://www.dataversity.net/acid-vs-base-the-shifting-ph-of-database-transaction-processing/)
```

Comment:
I have no idea why, but "https" is not appearing in the link text:

![image](https://user-images.githubusercontent.com/10886961/50783211-0402db80-125f-11e9-9990-891dd710d1fc.png)

Perhaps use \*\<https://www.dataversity.net/acid-vs-base-the-shifting-ph-of-database-transaction-processing/>\* so that the link text corresponds to the link?